src/VisualStudio/VisualBasic/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -167,7 +167,21 @@
                         Content="{x:Static local:AdvancedOptionPageStrings.Option_Colorize_regular_expressions}" />
                     <CheckBox x:Name="Report_invalid_regular_expressions"
                         Content="{x:Static local:AdvancedOptionPageStrings.Option_Report_invalid_regular_expressions}" />
-                    <CheckBox x:Name="Highlight_related_components_under_cursor"
+                    <CheckBox x:Name="Highlight_related_regular_expression_components_under_cursor"
+                        Content="{x:Static local:AdvancedOptionPageStrings.Option_Highlight_related_components_under_cursor}" />
+                </StackPanel>
+            </GroupBox>
+
+            <GroupBox x:Uid="JsonGroupBox"
```

Comment:
We really should unify this with the C# side to avoid the duplication. Out of scope for this PR but something to think about.